sorc/emcsfc_ice_blend.fd/emcsfc_ice_blend.f90

```diff
@@ -87,6 +87,15 @@
 !!
 !! In the SH, the blended value is simply the 5-minute ice concentration
 !! at 'water' points.  'Coast' and 'land' points are bitmapped out.
+!! @param j search position of a file
+!! @param lugi set to 0 - no grib index file
+!! @param jdisc set to 2 - search for discipline
+!! @param jpdtn search for product definition template number
+!! @param jgdtn search for grid definition template number; 0 - lat/lon grid
+!! @param jids array of values in identification section, set to wildcard
+!! @param jgdt array of values in grid definition template 3.m
+!! @param jpdt array of values in product definition template 4.n
```

Comment:
Well, we're back to where we started. ;-)

These values are not parameters so they should not be documented with the @param tag.

Instead, these comments should be applied to the variables defined below.

Do that by using the '!<' doxygen comment marker, like this:

`integer :: some_variable !< Description of that variable.
`

So move the comments to the variables, and get rid of the param statements.

Reply:
I understood from your reply above as your example. I modified it again hope to in-line with yours this time.